org-jira.el

```diff
@@ -1020,6 +1020,12 @@ See`org-jira-get-issue-list'"
    (org-jira-refresh-issue)))
 
 
+(defun get-id-name-alist (name ids-to-names)
```

Comment:
Good abstraction, but could you name it with the org-jira prefix?  So `(defun org-jira-get-id-name-alist` (also add a docstring in format that makes flycheck happy, something like `"Returns an alist containing id and name keys, generated from NAME and IDS-TO-NAMES."` (that could probably be worded better)